KH3Randomizer/Pages/Index.razor

```diff
@@ -88,37 +88,17 @@
     private List<Tuple<Option, Option>> modifications = new();
 
     private string hintSelectionType = "Verbose";
-    private Dictionary<string, Dictionary<string, bool>> availableHints = new Dictionary<string, Dictionary<string, bool>> {
-        { "Proofs", new Dictionary<string, bool>
-        { { "Proof of Promises", true }, { "Proof of Times Past", true }, { "Proof of Fantasy", true } }
-        },
-        { "Abilities", new Dictionary<string, bool>
-        { { "Pole Spin", true }, { "Block", true }, { "Dodge Roll", true }, { "Air-Slide", true },
-              { "Double Flight", true }, { "Second Chance", true }, { "Withstand Combo", true }, }
-        },
-        { "Heartbinders", new Dictionary<string, bool>
-        { { "Dream Heartbinder", true }, { "Pixel Heartbinder", true }, { "Pride Heartbinder", true },
-              { "Ocean Heartbinder", true }, { "\'Ohana Heartbinder", true } }
-        },
-        { "Magic", new Dictionary<string, bool>
-        { { "Fire", true }, { "Fira", true }, { "Firaga", true },
-              { "Water", true }, { "Watera", true }, { "Waterga", true },
-              { "Cure", true }, { "Cura", true }, { "Curaga", true },
-              { "Blizzard", true }, { "Blizzara", true }, { "Blizzaga", true },
-              { "Thunder", true }, { "Thundara", true }, { "Thundaga", true },
-              { "Aero", true }, { "Aerora", true }, { "Aeroga", true } }
-        },
-        { "Formchanges", new Dictionary<string, bool>
-        { { "Hero\'s Origin", true }, { "Grand Chef", true }, { "Ultima Weapon", true },
-              { "Oblivion", true }, { "Oathkeeper", true } }
-        },
-        { "Secret Reports", new Dictionary<string, bool>
-        { { "Secret Report 1", true }, { "Secret Report 2", true }, { "Secret Report 3", true }, { "Secret Report 4", true },
-              { "Secret Report 5", true }, { "Secret Report 6", true }, { "Secret Report 7", true }, { "Secret Report 8", true },
-              { "Secret Report 9", true }, { "Secret Report 10", true }, { "Secret Report 11", true }, { "Secret Report 12", true },
-              { "Secret Report 13", true }
-            }
-        },
+    private List<string> importantChecks = new List<string>()
+    {
+        "Proof of Promises", "Proof of Times Past", "Proof of Fantasy",
+        "Secret Report 1", "Secret Report 2", "Secret Report 3", "Secret Report 4",
+        "Secret Report 5", "Secret Report 6", "Secret Report 7", "Secret Report 8",
+        "Secret Report 9", "Secret Report 10", "Secret Report 11", "Secret Report 12", "Secret Report 13",
+        "Magic: Aero", "Magic: Blizzard", "Magic: Cure",
+        "Magic: Fire", "Magic: Thunder", "Magic: Water",
+        "\'Ohana Heartbinder", "Dream Heartbinder", "Ocean Heartbinder",
+        "Pixel Heartbinder", "Pride Heartbinder",
+        "Ability: Pole Spin"
```

Comment:
I believe people will still want Second Chance, Withstand Combo and Block included in important checks.

Reply:
Added Block, Second Chance, and Withstand Combo to the default list of important checks. Went ahead and added Air Slide and Dodge Roll as well since a lot of people have been talking about those defensives/movement options